packages/catalog-generator/pom.xml

```diff
@@ -160,6 +160,26 @@
           </classPatterns>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>${version.maven-enforcer-plugin}</version>
+        <executions>
+          <execution>
+            <id>enforce-java-version</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireJavaVersion>
+                  <version>[17]</version>
```

Comment:
not `[17]` but `17`

Reply:
or `[17,)` if you prefer to be more explicit and do no tuse the short notation
![image](https://github.com/user-attachments/assets/0632a63e-3fdb-440a-a8f4-67fd5d4dab60)